packages/utils/src/lexicographicalStringComparer.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Given 2 strings, it determines their relative ordering by comparing their lexicographical (dictionary) order.
+ * It returns a numeric value
+ *
+ * 1: If string a is lexicographically greater than string b.
+ * -1: If string b is lexicographically greater than string a.
+ * 0: If both input strings a and b are lexicographically equal.
+ * **/
+const lexicographicalStringComparer = (a: string, b: string) => {
```

Comment:
@jim-deriv I wonder if we can use native .localCompare() method instead of this function? :) Like this:
a.localeCompare(b);

Reply:
@maryia-deriv,  I'm not sure why in the original implementation they decided not to use `.localCompare()`, but I will replace it with the existing code. Though I'm usually hesitant to change existing code when I'm refactoring, unless I really have to